### ISSUE TITLE
MSAA not working; FXAA wasn't working

### ISSUE THREAD

**Issue by nagusameruru** [TekkenOverlay/TekkenOverlay#179]
When changing AA methods, MSAA is visually the same as None. This happens at all AA qualities within the Tekken 8 graphics menu. TAA, TSR, and FXAA all make differences; however, FXAA was not making changes at first. Honestly, I'm not sure why that changed and I haven't been able to replicate it. Will update if I find the cause for that.

Off:
![image](https://github.com/TekkenOverlay/TekkenOverlay/assets/76086075/40a0dd8d-c173-44bd-9236-58932359431d)
MSAA:
![image](https://github.com/TekkenOverlay/TekkenOverlay/assets/76086075/c0d78674-22e8-4e1c-8563-8ceb973688fd)
TAA:
![image](https://github.com/TekkenOverlay/TekkenOverlay/assets/76086075/99e9b803-67de-4e21-8969-adee543a8f5a)

Tekken 8 and Tekken Overlay settings:
![image](https://github.com/TekkenOverlay/TekkenOverlay/assets/76086075/17f67063-ae5a-47e4-b525-f90145889332)

**Comment by YelovSK** [comment]
Even if MSAA was working, it would not be worth using, as I don't think it works on stuff like hair. It relies on temporal AA techniques. But I also had a similar question here #175, as I'm not sure if the overlay settings override the upscaling method selected in the in-game menu. I think only the internal resolution is used, so in your screenshot it would use 75% render scale, but use the AA method selected in the tekken overlay options.

**Comment by xNadeemx** [comment]
Yeah MSAA wouldn't work for a lot of the hair since it wont fix the dithering and blend all the dots together, I do recommend if on Nvidia to use DLDSR in the nvidia control panel and then DLSS + DLSS tweaks to adjust the scaling percentage for the best effect. Set them all to preset C btw.
